src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -1,7 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- The documentation in here must be kept up to date with that embedded in comments in newrelic.config -->
-<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" targetNamespace="urn:newrelic-config" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:nr="Configuration.xsd">
-  <xs:element name="configuration">
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:tns="urn:newrelic-config" targetNamespace="urn:newrelic-config" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:nr="Configuration.xsd">
+
+    <xs:complexType name="ErrorClassType">
+        <xs:annotation>
+            <xs:documentation>
+                A list of fully qualified error class names to be marked as ignored/expected.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence minOccurs="0" maxOccurs="unbounded">
+            <xs:element name="errorClass" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation>
+                        Fully qualified class name of an exception, such as "System.IO.FileNotFoundException".
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="ErrorMessagesType">
```

Comment:
Should this be something like `ErrorMessagesMap` or `ErrorMessagesCollection` to better represent that this type defines a collection of error messages.